Op bunching 2: Bunch contiguous ops for data store in a batch - data store part (#22840)
microsoft/FluidFramework: .changeset/hot-falcons-grab.md

---
"@fluidframework/datastore-definitions": minor
"@fluidframework/runtime-definitions": minor
---
---
"section": other
---

> The timing of the `op` event on `IFluidDataStoreRuntimeEvents` and `IContainerRuntimeBaseEvents` has changed.

Reviewer: The headers/titles (first line of the changeset contents) usually skip the trailing stop. The rephrasing is pretty subjective, it just felt more "title-y" to me. Feel free to keep as is and just remove the period.
```suggestion
Change when the `op` event on `IFluidDataStoreRuntimeEvents` and `IContainerRuntimeBaseEvents` is emitted
```
